pkg/services/provisioning/dashboards/file_reader.go

```diff
@@ -47,15 +47,25 @@ func NewDashboardFileReader(cfg *DashboardsAsConfig, log log.Logger) (*fileReade
 		log.Error("Cannot read directory", "error", err)
 	}
 
-	absPath, err := filepath.Abs(path)
+	copy := path
+	path, err := filepath.Abs(path)
 	if err != nil {
 		log.Error("Could not create absolute path ", "path", path)
-		absPath = path //if .Abs return an error we fallback to path
+	}
+
+	path, err = filepath.EvalSymlinks(path)
```

Comment:
Shouldn't we run `filepath.Abs(path)` again after evaluating any symbolic links given that EvalSymlinks gives a relative path if the linked path is relative?